optimism/inverse/test/FiniteDifferenceFixture.py

```diff
@@ -0,0 +1,67 @@
+from optimism.test.MeshFixture import *
```

Comment:
May I ask that you import the module directly and avoid `import *` in general? This pulls everythin into the top-level namespace makes it harder to see where things come from and to avoid name clashes. I realize that this idiom is used throughout optimism, but I've been trying to remove them bit by bit.